libevmasm/Instruction.h

```diff
@@ -186,6 +186,9 @@ enum class Instruction: uint8_t
 
 	RJUMP = 0xe0,             ///< relative jump
 	RJUMPI = 0xe1,            ///< conditional relative jump
+	CALLF = 0xe3,             ///< call function in a EOF code section
+	RETF = 0xe4,              ///< return to caller from the code section of EOF continer
+	JUMPF = 0xe5,             ///< jump to a code section of EOF contaner. No stack cleaning.
```

Comment:
Isn't stack cleaning only done when returning? Does not seem relevant here.

```suggestion
	JUMPF = 0xe5,             ///< jump to a code section of EOF contaner without adding a new return stack frame.
```

Reply:
Also, you have some typos here: `contaner`, `continer` :)

Reply:
`continer` is still there. Also `can cantinue`, `Atempted` and `call call section 0` :)

Our codespell job should have detected and reported them though. It seems that it needs an updated dictionary. After we merge #15589, it should be working again.